packages/labs/ssr/src/lib/render-lit-html.ts

```diff
@@ -184,6 +184,20 @@ type CustomElementClosedOp = {
   type: 'custom-element-close';
 };
 
+/**
+ * Operation to possibly emit the `<!--lit-node-->` marker; the operation
+ * always emits if there were attribtue parts, and may emit if the node
+ * was a custom element and it needed `defer-hydration` because it was
+ * rendered in the shadow root of another custom element host; we don't
```

Comment:
Why don't we know this? It's unclear given that `customElementHostStack` is maintained.

Reply:
Answer: because this is template based and not instance based.